lib/client-assets.php

```diff
@@ -1163,10 +1113,10 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 	$meta_box_url = admin_url( 'post.php' );
 	$meta_box_url = add_query_arg(
 		array(
-			'post'           => $post->ID,
-			'action'         => 'edit',
-			'classic-editor' => true,
-			'meta_box'       => true,
+			'post'            => $post->ID,
+			'action'          => 'edit',
+			'meta-box-loader' => true,
+			'_wpnonce'        => wp_create_nonce( 'meta-box-loader' ),
```

Comment:
Is this to match the URL used in Core?

Do you think we could remove bootstraping Gutenberg entirely in the plugin (and just upgrade the scripts)

Reply:
>Do you think we could remove bootstraping Gutenberg entirely in the plugin (and just upgrade the scripts)

Yes, in fact, what I've done here is simply to align the bootstrapping to exactly what core does, in order to facilitate it being dropped altogether.

https://github.com/WordPress/wordpress-develop/blob/8b4330bb6ea196005b5f0f84070b64c139f39b37/src/wp-admin/edit-form-blocks.php#L115-L126